tests/rados/test_osd_ecpool_inconsistency_scenario.py

```diff
@@ -356,7 +334,12 @@ def run(ceph_cluster, **kw):
         except Exception as e:
             log.info(e)
             return 1
+        msg_obj_count = f"Scenario-6:The object count value is-{obj_count}"
+        log.info(msg_obj_count)
         scrub_obj.set_osd_flags("unset", "noscrub")
```

Comment:
I believe that the noscrub should only be unset at the end of the workflow.

Reply:
Before unsetting the noscrub flag I am getting the inconsistent object count.The results are getting at  Line#331. After  this line I am verifying the count. To avoid the confusion I will move the line to the end.